docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -92,63 +96,159 @@ You can also build the collector-contrib repository by cloning it and running:
 
-[source,yml]
+[source,bash]
 ----
-exporters:
-  elastic:
-    apm_server_url: "https://elasticapm.example.com"
+export OTEL_RESOURCE_ATTRIBUTES=service.name=frontend,service.version=1.0-SNAPSHOT,deployment.environment=staging
+java -javaagent:/path/to/opentelemetry-javaagent-all-0.10.1.jar \
+     -Dotel.otlp.endpoint=my-otel-collector.mycompany.com:55680 \
+     -jar target/frontend-1.0-SNAPSHOT.jar
 ----
 
-See the <<open-telemetry-elastic-config-ref,configuration reference>> for additional configuration options,
-like specifying an API key, secret token, or TLS settings.
+Here is an example of how to capture business metrics from an application.
 
-The Elastic exporter must also be defined in `service.pipelines.traces.exporters`.
-For example:
-
-[source,yml]
+[source,java]
 ----
-service:
-  pipelines:
-    traces:
-      exporters: [elastic]
+// initialize metric
+Meter meter = OpenTelemetry.getGlobalMeter("my-frontend");
+DoubleCounter orderValueCounter = meter.doubleCounterBuilder("order_value").build();
+
+public void createOrder(HttpServletRequest request) {
+
+   // create order in the database
+   ...
+   // increment business metrics for monitoring
+   orderValueCounter.add(orderPrice);
+}
 ----
 
-If we put everything together, here's an example configuration file that accepts input from an OpenTelemetry Agent,
-processes the data, and sends it to an {ess} instance.
+[float]
+[[open-telemetry-elastic-configure]]
+==== Configure the collector
+
+Create a `yaml` configuration file.
+
+This example configuration file accepts input from an OpenTelemetry Agent, processes the data, and sends it to an {ess} instance.
 
 [source,yml]
 ----
-receivers:
-  otlp:
-    endpoint: localhost:55680
-processors:
-  batch:
-    timeout: 1s
-    send_batch_size: 1024
-exporters:
-  elastic:
-    apm_server_url: "https://elasticapm.example.com"
-    secret_token: "ESS_TOKEN"
-service:
-  pipelines:
-    traces:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [elastic]
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+          endpoint: 'localhost:55680'
+    hostmetrics: <1>
+      collection_interval: 1m
+      scrapers:
+        load:
+        memory:
+
+  processors:
+    batch: null
+    queued_retry: null
+
+  exporters:
+    elastic:
+      apm_server_url: 'https://elasticapm.example.com' <2>
+      secret_token: 'APM_SERVER_SECRET_TOKEN'
+
+  service:
+    pipelines:
+      metrics:
+        receivers:
+          - otlp
+          - hostmetrics
+        exporters:
+          - elastic <3>
+      traces:
+        receivers:
+          - otlp
+        processors:
+          - batch
+        exporters:
+          - elastic <4>
 ----
+<1> The `hostmetrics` receiver must be defined to generate metrics about the host system scraped from various sources.
+<2> At a minimum, you must define the URL of the APM Server instance you are sending data to. See the <<open-telemetry-elastic-config-ref,configuration reference>>
```

Comment:
Matching the other callouts

```suggestion
<2> At a minimum, you must define the URL of the APM Server instance you are sending data to. See the <<open-telemetry-elastic-config-ref,configuration reference>>.
```

Reply:
The end of the sentence is on the next line.

Reply:
lol. Oops!